test/e2e/eventListener/eventListener_test.go

```diff
@@ -0,0 +1,63 @@
+// +build e2e
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipeline
+
+import (
+	"testing"
+
+	"github.com/tektoncd/cli/test/builder"
+	"github.com/tektoncd/cli/test/cli"
+	"github.com/tektoncd/cli/test/framework"
+	"github.com/tektoncd/cli/test/helper"
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/icmd"
+	knativetest "knative.dev/pkg/test"
+)
+
+const (
+	teEventListenerName = "listener"
+)
+
+func TestEventListenerE2E(t *testing.T) {
+	t.Parallel()
+	c, namespace := framework.Setup(t)
+	knativetest.CleanupOnInterrupt(func() { framework.TearDown(t, c, namespace) }, t.Logf)
+	defer framework.TearDown(t, c, namespace)
+
+	kubectl := cli.NewKubectl(namespace)
+	tkn, err := cli.NewTknRunner(namespace)
+	assert.NilError(t, err)
+
+	t.Logf("Creating eventlistener in namespace: %s", namespace)
+	kubectl.MustSucceed(t, "create", "-f", helper.GetResourcePath("eventlistener.yaml"))
+
+	t.Run("Assert if eventlistener status to be true", func(t *testing.T) {
+		res := tkn.Run("eventlistener", "list", `-o=jsonpath={range.items[*]}{.metadata.name}{"\n"}{end}`)
+		expected := builder.ListResourceNamesForJSONPath(
+			builder.GetEventListenerListWithTestData(t, c,
+				map[int]interface{}{
+					0: &builder.EventListenerListData{
+						Name:   teEventListenerName,
+						Status: "True",
```

Comment:
What this test should assert is whether the list command contains the `AVAILABLE` column and that the `AVAILABLE` value is true since this is how the list command will display the value to end users.

Reply:
I updated the code.